### PR TITLE
:bug: fix ip-adapter controlnet img2img missing code

### PR DESCRIPTION
**Pull request by Honey-666**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

hi
@yiyixuxu 
#6516
I added code to it

### CODE REVIEW

**Review by yiyixuxu — APPROVED** [review]
thanks!

File: tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py
Comment (sayakpaul) [inline tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py:404]: Do we need to add the slow test? @yiyixuxu

**Review by sayakpaul — APPROVED** [review]
Barring the question on slow test, I am good with the PR. Good job!

File: tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py
Comment (yiyixuxu) [reply tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py:404]: we don't need to - do we prefer not too?

**Comment by yiyixuxu** [comment]
@Honey-666 
can we get the quality test pass? 
did you run `make style` and `make fix-copies`?

**Comment by Honey-666** [comment]
> @Honey-666 can we get the quality test pass? did you run `make style` and `make fix-copies`?

Oh, I'm sorry I overlooked it. I will try to resubmit it

**Comment by yiyixuxu** [comment]
Ohh great! 
I guess we only have one more question left here before we can merge https://github.com/huggingface/diffusers/pull/6528#discussion_r1449775034

I don't think we need a slow test here, but I also would not mind to add it. 
cc @sayakpaul here again - do you prefer to not have a test here? 

YiYi

**Comment by sayakpaul** [comment]
I prefer not to have a slow test here, yet, yes.

**Review by patrickvonplaten — APPROVED** [review]
Good from my side with and without the slow test!

**Comment by yiyixuxu** [comment]
@Honey-666 
let's remove the slow tests to make @sayakpaul happy :) 
will merge after that

**Comment by Honey-666** [comment]
> @Honey-666 let's remove the slow tests to make @sayakpaul happy :) will merge after that

Ok 
Do I just need to remove the test code from the `test_ip_adapter_stable_diffusion.py` file?

**Comment by yiyixuxu** [comment]
@Honey-666 yes :)

**Comment by Honey-666** [comment]
> @Honey-666 yes :)

Thanks a million

**Pull request merged**
